http/reactive-routes/src/test/java/io/quarkus/ts/BasicsRouteHandlerIT.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.ts;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class BasicsRouteHandlerIT {
+
+    @QuarkusApplication(ssl = true)
```

Comment:
why ssl true?

Reply:
I think that was something that was left over. In this scenario we are not testing anything related to http2 so..I think that could be removed